docs/resources/item.md

```diff
@@ -0,0 +1,127 @@
+---
+page_title: "onepassword_item Resource - terraform-provider-onepassword"
+subcategory: ""
+description: |-
+  A 1Password item.
+---
+
+# Resource `onepassword_item`
+
+A 1Password item.
+
+## Example Usage
+
+```terraform
+resource "onepassword_item" "demo_password" {
+  vault = var.demo_vault
+
+  title    = "Demo Password Recipe"
+  category = "password"
+
+  password_recipe {
+    length  = 40
+    symbols = false
+  }
+}
+
+resource "onepassword_item" "demo_login" {
+  vault = var.demo_vault
+
+  title    = "Demo Terraform Login"
+  category = "login"
+  username = "test@example.com"
+}
+
+resource "onepassword_item" "demo_db" {
+  vault    = var.demo_vault
+  category = "database"
+  type     = "mysql"
+
+  title    = "Demo TF Database"
+  username = "root"
+
+  database = "Example MySQL Instance"
+  hostname = "localhost"
+  port     = 3306
+}
+```
+
+## Schema
+
+### Required
+
+- **vault** (String, Required) The UUID of the vault the item is in.
+
+### Optional
+
+- **category** (String, Optional) The category of the item.
```

Comment:
Would it be possible to list all options or link to a list of options?

Reply:
Good point. I added the validation but it doesn't appear to pull that in. I will update the Description to include the validation